mautic/mautic
# Cancelling new campaign should not give 500 error.

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ 4.1 ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [x] 
| Issue(s) addressed                     | Fixes #11181 



#### Description:



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Go to campaigns and click "New"
3. Cancel campaign
4. Campaign list page should be shown instead of 500 Error.




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11348"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
